packages/http-client-java/generator/http-client-generator/src/main/java/com/microsoft/typespec/http/client/generator/fluent/TypeSpecFluentPlugin.java

```diff
@@ -88,6 +88,9 @@ public TypeSpecFluentPlugin(EmitterOptions options, boolean sdkIntegration) {
         if (options.getGenerateAsyncMethods() != null) {
             SETTINGS_MAP.put("generate-async-methods", options.getGenerateAsyncMethods());
         }
+        if (options.getPropertyIncludeAlways() != null) {
+            SETTINGS_MAP.put("property-include-always", options.getPropertyIncludeAlways());
```

Comment:
Could we add some comment to explain the flag? e.g. add the loadtesting example mentioned in [issue](https://github.com/Azure/autorest.java/issues/3102). Currently it is not easy to understand what the flag is doing by looking at the name `property-include-always`.